.github/workflows/build.yml

```diff
@@ -39,15 +30,15 @@ jobs:
         with:
           name: nixpkgs-terraform
           authToken: ${{ secrets.CACHIX_AUTH_TOKEN }}
-          skipPush: false
```

Comment:
I think this change was just used for testing, should we set it back to `false`?

Reply:
@oscar-izval since we are using `cachix watch-exec` to gradually push new entries on the Nix store, I set it to `true`.

Reply:
That's right, I forgot sorry!